app/src/main/java/org/metabrainz/android/presentation/features/userprofile/ProfileTheme.kt

```diff
@@ -0,0 +1,45 @@
+package org.metabrainz.android.presentation.features.userprofile
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.MaterialTheme.shapes
+import androidx.compose.material.MaterialTheme.typography
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+
+//Colors
+val Appbg = Color(0xFF263238)
+val AppbgLight = Color(0xFFe0e0e0)
+val EnabledChipColor = Color(0xFFccabff)
+val DisabledChipColor = Color(0xFFd8cfe8)
+val ChipTextColor = Color(0xFF323232)
+
+private val DarkColorPalette = darkColors(
+    surface = Appbg
+)
+
+private val LightColorPalette = lightColors(
+    surface = AppbgLight
+)
+
+@Composable
+fun ProfileTheme(
```

Comment:
Do we have to define themes for every activity separately or we can prefer having global themes? You could do some research on this and let me know :)